app/ts/components/requirements.tsx

```diff
@@ -163,6 +192,14 @@ export const Create = ( { contentHashInput, resolutionAddressInput, loadingInfos
 		return false
 	})
 
+	const twoLevelDomainExpiryBiggerThanLowestLevelExpiry = computed(() => {
```

Comment:
🤔  We could have a naming convention like `level2Domain...`.  Would make it so levels are always just a number, and reads a bit smoother than `twoLevelDomain` (which sounds like you have a domain with two levels).